front/app/components/admin/PostManager/components/FilterSidebar/topics/FilterSidebarTopics.tsx

```diff
@@ -3,15 +3,16 @@ import React, { MouseEvent } from 'react';
 import { Box, Text } from '@citizenlab/cl2-component-library';
 import { xor } from 'lodash-es';
```

Comment:
Just implemented the same solution that you did for the FilterSidebarPhases :]

Reply:
Great solution! 😄